catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/CachedResourceMetacardComparator.java

```diff
@@ -52,35 +49,18 @@
   private static final List<Function<Metacard, ?>> METACARD_METHODS =
       ImmutableList.of(
           Metacard::getModifiedDate,
-          Metacard::getThumbnail,
-          Metacard::getResourceURI,
-          Metacard::getLocation,
-          Metacard::getEffectiveDate,
           Metacard::getCreatedDate,
-          Metacard::getExpirationDate,
-          Metacard::getContentTypeName);
-
-  private static final Set<String> ATTRIBUTES_TO_IGNORE =
-      ImmutableSet.of(
-          Metacard.CHECKSUM,
-          Metacard.MODIFIED,
-          Metacard.RESOURCE_SIZE,
-          Metacard.THUMBNAIL,
-          Metacard.RESOURCE_URI,
-          Metacard.GEOGRAPHY,
-          Metacard.EFFECTIVE,
-          Metacard.CREATED,
-          Metacard.EXPIRATION,
-          Metacard.CONTENT_TYPE);
+          (m) -> m.getAttribute(Core.METACARD_MODIFIED),
```

Comment:
❓ Since we need to use `getAttribute()` for `METACARD_MODIFIED` and `METACARD_CREATED`, think we should just use that for `MODIFIED` and `CREATED` as well and just have the list contain attribute names instead of method references? That would make things consistent and would allow us to print the attribute name in the logging statement here: https://github.com/codice/ddf/blob/e85c0e0a8ae53c657a85bb4092d04becf3bc255c/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/CachedResourceMetacardComparator.java#L108-L111.

Reply:
fixed